docs/analyzer/checker_and_analyzer_configuration.md

```diff
@@ -70,16 +70,19 @@ If they are needed, they can be switched on using the following command
 is for example `-Xclang -analyzer-checker=debug.ExprInspection`.
 
 ## Z3 Theorem Prover
-The static analyzer supports using the
+The _Clang Static Analyzer_ supports using the
 [Z3 Theorem Prover](https://github.com/Z3Prover/z3) from Microsoft Research as
 an external constraint solver. This allows reasoning over more complex queries,
-but performance is `~15x` slower than the default range-based constraint
-solver. To enable the Z3 solver backend, Clang must be built with the
-`CLANG_ANALYZER_BUILD_Z3=ON` option, and the
+but performance is expected to be **15-20 times** slower than the default
+range-based constraint solver engine.
+
+To enable the Z3 solver backend, Clang must be built with the
+`LLVM_ENABLE_Z3_SOLVER=ON` compile-time option (for versions earlier than
+**9.0**, `CLANG_ANALYZER_BUILD_Z3=ON` must be used instead!), and the
```

Comment:
My understanding is that since 9.0 LLVM must use the option …, while before 9.0 clang must use the opiton ….

The text suggests that in old and new version the flag must be passed to the clang build configuration.

Reply:
LLVM has been retrofitted with a monorepo structure even for the old (originally pre-monorepo) versions. So there is only one build that you do, for the entirety of the LLVM umbrella project and you have to pass this flag there.

Reply:
This discussion is a very small nuance, so the suggested text is good enough, as it is.